applications/luci-app-cloudflared/root/usr/share/rpcd/acl.d/luci-app-cloudflared.json

```diff
@@ -0,0 +1,15 @@
+{
+	"luci-app-cloudflared": {
+		"description": "Grant access to Cloudflared configuration",
+		"read": {
+			"file": [ "/var/log/cloudflared.log" ],
+			"ubus": {
+				"service": [ "list" ]
+			},
+			"uci": [ "cloudflared" ]
+		},
+		"write": {
```

Comment:
*hint*
```json
		"write": {
			"uci": ["cloudflared"],
			"file": {
				"/etc/cloudflared/*": ["read"],
				"/etc/cloudflared/config.yml": ["write"],
				"/etc/cloudflared/config.yaml": ["write"],
				"/etc/cloudflared/cert.pem": ["write"],
			}
```

Reply:
It would be simpler to make `write` for the whole `/etc/cloudflared`

Reply:
I made the whole /etc/cloudflared writable